### PR TITLE
User/karayanni/ui testing

### PR DESCRIPTION
**Pull request by jerieskhoury**
i added some field to the ui pages and renovated them a little bit

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import Board as b
 import  destinations as d
 import  cards as c
+import cgi
```
Comment (karayanni) [inline game.py:4]: what are those for?

File: templates/about.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+  </head>
+  
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <a class="navbar-brand" href="/nader">nader</a>
```
Comment (karayanni) [inline templates/about.html:15]: consider removing/changing the Nader bar

File: templates/home.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+
+    <title>Hello, world!</title>
```
Comment (karayanni) [inline templates/home.html:11]: consider removing

File: game.py (hunk 1 above)
Comment (jerieskhoury) [reply game.py:4]: testing something forgot to remove

**Pull request merged**